src/libserver/re_cache.c

```diff
@@ -126,7 +126,6 @@ struct rspamd_re_cache {
 	lua_State *L;
 #ifdef WITH_HYPERSCAN
 	enum rspamd_hyperscan_status hyperscan_loaded;
-	gboolean disable_hyperscan;
```

Comment:
This should not be removed - it is an used feature to disable HS from the configuration.